src/main/java/org/folio/repository/tag/TagRepositoryImpl.java

```diff
@@ -100,6 +101,22 @@ public TagRepositoryImpl(Vertx vertx) {
     return mapResult(resultSetFuture, this::readTags);
   }
 
+  @Override
+  public CompletableFuture<List<Tag>> findByRecordByIds(String tenantId, List<String> recordIds, RecordType recordType) {
+
+    String placeholders = createPlaceholders(recordIds.size());
+    recordIds.add(recordType.getValue());
```

Comment:
Please avoid incoming list modification. Treat it is read only

Reply:
Thanks, @dmtkachenko . I will fix it.